blaze/utils.py

```diff
@@ -292,7 +300,7 @@ def numpy_pandas_function_from_str(f):
         mod = pd
 
     else:
-        raise NotImplementedError("accepts numpy/pandas/builtin funcs only")
+        raise wz_ex.NotImplemented("accepts numpy/pandas/builtin funcs only")
 
```

Comment:
Raising http exceptions from `werkzeug.exceptions`, trying to be more specific; however, upon POST, the server will invoke [`check_request`](https://github.com/blaze/blaze/blob/master/blaze/server/server.py#L511) to see if the serialized data loads correctly. The `check_request` function expects [ValueError and traps it](https://github.com/blaze/blaze/blob/master/blaze/server/server.py#L215)

So, perhaps utils.py should only raise ValueError

@kwmsmith @llllllllll